Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
```

Comment:
<p>same comment as for the block above, the error message looks quite self-explanatory and it breaks the flow of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>As above.</p>